Move "new file" WordPress folder to PHP category

Chris2011 opened the pull request:
At the moment calling the "new file" wizard will show WordPress folder on each project type, which doesn't make sense. So I added the specific category where it belongs to, which is "PHP" in this case. So it will only show up in php projects. Also I moved the WordPress folder to the PHP folder.

Chris2011 committed:
#70 - Added Go To Declaration for methods inside strings of add_shortcode, register_activation_hook and register_deactivation_hook.

Chris2011 committed:
#70 - Updated readme, added missing Hyperling navigation functions.

Commit:
Merge branch 'master' of https://github.com/junichi11/netbeans-wordpress-plugin

junichi11 commented on src/org/netbeans/modules/php/wordpress/resources/templates/package-info.java:
```diff
@@ -7,35 +7,40 @@
     @TemplateRegistration(
-            folder = "WordPress",
+            folder = "Scripting/WordPress",
+            category = "PHP",
```
OK.

junichi11 commented on src/org/netbeans/modules/php/wordpress/resources/templates/package-info.java:
```diff
@@ -7,35 +7,40 @@
             scriptEngine = "freemarker"),
     @TemplateRegistration(
-            folder = "WordPress",
+            folder = "Scripting/WordPress",
```
I prefer "WordPress". (Please don't change it.)

Chris2011 replied:
I would say this has less to do with preferences. WordPress belongs to PHP. We also have it inside the global options -> PHP -> Frameworks & Tools and also inside the project properties -> Frameworks. At the end I don't care about if you really want to change this back but as I said, it belongs to it and should be there too.

junichi11 replied:
Category is already PHP by your below change, right?
If we add the folder, we have to click the folder once every time. I don't want to do it.

Chris2011 replied:
Yes, you are right, it is already for this category. I will change the other thing back. Thx for your info.

Chris2011 replied:
Fixed

junichi11 replied:
OK.

junichi11 commented:
Could you squash your changes with the proper comment as one commit?

Chris2011 committed:
Move WordPress folder to category PHP where it belongs to

At the moment calling the "new file" wizard will show WordPress folder on each project type, which doesn't make sense. So I added the specific category where it belongs to, which is "PHP" in this case. So it will only show up in php projects.

Chris2011 commented:
Finally done. I dunno why github shows old commits that are already on master.

Chris2011 commented:
At the end I'm fine to checkout master, rename the old branch create a new one with the same name and add my changes again to make it clean. What do you think?

junichi11 commented:
> At the end I'm fine to checkout master, rename the old branch create a new one with the same name and add my changes again to make it clean. What do you think?

Looks good to me. Please do it. 
Please change the base branch to `nb180`.
